wiki/Tournaments/oPHST/2018/en.md

```diff
@@ -120,6 +119,14 @@ Mappools will be revealed one week in advance of each stage.
 
 - The tournament shall host the following game modes: osu!standard, osu!catch, osu!mania 4K, and osu!mania 7K. In addition, the tournament shall use **Score v2** as its scoring format. This requires players to play on the following osu! Builds: Stable (Latest), Beta, and Cutting Edge. 
 
+### Tournament Formats
+
+- The **osu!standard** division will adopt a Qualifier Round (explained below) in order to reduce the field into 32 players which will then be subjected to a **single elimination** bracket. Seeding for this division will be based on qualifier results.
+
+- The **osu!catch** division will be subjected to a **16-player double elimination** bracket. Seeding for this division will be based on their Global Ranking as of April 15, 2018, 18:00 UTC+8.
+
+- Both **osu!mania 4K** and **osu!mania 7K** divisions shall adopt a Qualifier Round (explained below) in order to reduce the field as registration numbers may see fit. If registrants exceed 32 players, a play-off round will be introduced in order to reduce the field to 16 players. The players are then split into four **single round robin** groups of four players each, the top 2 of each group will advance to a **double elimination** bracket. Seeding for this division will be based on qualifier results.

```

Comment:
(explained below) -> (explained above)

Reply:
shouldn't this be "below" since the Qualifier rules are a few subsections after this one?

Reply:
hm, ignore then